components/rhs_comment/rhs_comment.jsx

```diff
@@ -393,7 +401,7 @@ export default class RhsComment extends React.Component {
                             </div>
```

Comment:
Commenting on line above, which GitHub sadly constrains me from doing naturally -- but we now have a case where `userProfile` will be `null`. It's a touch weird to have that wrapped in `<strong>` tags, and I'm just wanting to make sure there isn't a case where the "spacing" required by the previously-always-rendered-userProfile is still required. Or is that handled by `col_name`?

Reply:
I remove things that aren't visible anyway, so, this texts shouldn't be affected, because when you try to render them, they aren't visible at all. In my environment is working as expected.